src/components/DashboardHeader/index.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { withStyles } from '@material-ui/core/styles';
+
+const styles = {
+  dashboardTitle: {
+    color: 'white',
+    backgroundColor: 'black',
+    borderBottom: '1px solid #fff',
+    padding: '0.35rem 0 0 1rem',
+  },
+  subtitle: {
+      color: '#d1d2d3',
+      fontWeight: '400',
+      margin: '.5rem .6rem .5rem',
+      padding: '.5em 0',
+  },
+};
+
+const DashboardHeader = ({ classes, title, subtitle }) => (
```

Comment:
Why does DashboardHeader exist?  Is there a plan to use it in other dashboards?  If this component has only one call point, I suggest it stay inlined.  At the very least, give it a name that indicates it is private (use underscore as prefix, like `_DashboardHeader`)